src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(toDo, number) {
+      this.result = toDo(this.result, number);
+
+      return this;
+    },
+
+    add(accum, number) {
+      return accum + number;
+    },
```

Comment:
1. you should work with one function parameter
2. use `this`

```suggestion
    add(value) {
      this.result += value;
      return this;
    },
```